docs/integrations/sumo_logic.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Sumo Logic Integration
+category: 62cdf6722c2c1602a4b69643
+hidden: false
+---
+
+# Sumo Logic Integration
+
+## Overview
+
+Phylum can be integrated with [Sumo Logic](https://www.sumologic.com) to incorporate Phylum's curated list of malicious open-source package hashes into a Sumo Logic lookup table for searching/matching of log data within Sumo Logic.
+
+## Getting Started
+
+Sumo Logic users who purchase a Phylum threat feed subscription can follow the instructions below to subscribe to Phylum's curated malicious package hash list distribution. Users will need to run a Python script to periodically load new data from Phylum's Threat Feed into Sumo Logic.
+
+There are two separate pieces of information needed to complete the subscription process:
+1. A Sumo Logic user account API token
+2. A Phylum user account API token
+
+### Obtain a Sumo Logic user account API token
+
+1. Log in to your Sumo Logic user dashboard and then click on your username (lower left), then `Preferences`.
+2. Add an `Access Key` under `My Access Keys`
+3. Copy the `Access ID` and `Access Key` and store them safely in a password manager.
+
+### Obtain a Phylum user account API token
+
+1. Follow the instructions [here](https://docs.phylum.io/docs/phylum_auth_create-token)
+
+### Install the Phylum app from Sumo Logic App Library
+After configuring collection you can [install](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) the Sumo Logic App for Phylum from App Catalog.
+
+### Install the Phylum Threat Feed Python Script
+Phylum provides a Python script to load threat feed data into Sumologic. This script should be run on a schedule to get regular updates from Phylum’s threat feed. 
+
+1. Clone the repository locally: `git clone https://github.com/SumoLogic/sumologic-public-partner-apps`
+1. Navigate to the Phylum subdirectory of the cloned repository, then to “script”: `cd Phylum/script`
+1. Install the required Python dependencies for the script: `pip install -r requirements.txt`
+1. Create a new lookup table in Sumo Logic under the User’s Personal folder: https://help.sumologic.com/docs/search/lookup-tables/create-lookup-table/
+    1. Note the path to the lookup table: An example path for a lookup table named “threat-table” in a folder named “phylum-threat-data” under the personal folder is: “/Library/Users/pete@phylum.com/phylum-threat-data/threat-table”
```

Comment:
```suggestion
    1. Note the path to the lookup table: An example path for a lookup table named “threat-table” in a folder named “phylum-threat-data” under the personal folder is: `/Library/Users/pete@phylum.com/phylum-threat-data/threat-table`
```

switching to avoid auto-linking as an email address